docs/create-handlers.md

```diff
@@ -3,14 +3,326 @@ id: create-handlers
+I usually directly create a script and make Nucleus-Coop set the instances for me. Recently, a software for automatically set Goldberg for you was just released if you still want to try manually https://www.reddit.com/r/GoldbergEmu/comments/kxsaki/goldberggui_frontend_for_goldberg_emulator/?utm_source=share&utm_medium=web2x&context=3
+
+
+
+### Scripting Tips
+
+If all work is time to script it. Now, You will start testing input restriction and focus methods!
+Start by checking how the public scripts work and use as an example one that uses the same game-engine of the game you are trying to script.
+##### Some tips and essential lines
+You can find most of the lines in the Mod-ReadMe.txt (inside Nucleus-Coop root folder) and in the Master_SCript.js (documentation link inside Reddit FAQ post).
+Also, all my public scripts (Bizzo) have descriptions inside that could help you.
+
+Here some tips and essential lines:
+
+#### Game Info
+***These lines must be inside the script:***
+Line  | Description
+----- | -----------
+Game.ExecutableName = "game.exe";  | 	What executable the game use. This will be used by Nucleus for "exe script selection", for run the game and as the process to follow for positioning and resizing.
+Game.GUID = "GameName";  | The name of the folder that will be created inside the Nucleus\content folder (Don't use spaces). In this folder, Nucleus will store the game files for each instance of the game.
+Game.GameName = "Game Name on Nucleu UI"; | Title of the game that will be shown on Nucleus.
+Game.MaxPlayersOneMonitor = 4; |	This is just an info. It will not limit the players' numbers.
+Game.MaxPlayers = 16; | 	This is just an info. Usually, we write the max players number the game support. 
+
+***Other important ones:***
+Line | Description
+---- | -----------
+Game.BinariesFolder = "Bin"; | In which folder the game exe is located? Relative path to where Nucleus should start the game's working folder to.
+Game.SteamID = "1234"; | Here the SteamID of the game. You can find it on https://steamdb.info/ . Add it only if the game is available on Steam.
+Game.LauncherTitle = ""; | The name of the launcher's window title. Some games need to go through a launcher to open. This is needed or else the application will lose the game's window.
+Game.LauncherExe = "gamelauncher.exe";  | If the game needs to go through a launcher before opening. Nucleus will use this exe to start the game and it will follow Game.ExecutableName process for resizing and positioning. You can also write it like this: Game.LauncherExe = "gamelauncher.exe|NucleusDefined"; for let the User select the launcher exe the very first time (useful if is not located in the game folder).
+Game.LauncherExeIgnoreFileCheck = true;	 | Do not check if Launcher Exe exists in game folder | you will need to provide a relative filepath from game root folder
+
+***Files copies lines***
```

Comment:
These look like a nice addition, but for now. Remove them, and add them in a new commit after instead

## Remove from here